decentraland/helpers/storage.js

```diff
@@ -0,0 +1,59 @@
+import { SecureStore } from "expo";
+import { createFromPrivateKey } from "tasit-account/dist/testHelpers/helpers";
+
+// Storage keys
+const EPHEMERAL_ACCOUNT_PRIV_KEY = "EPHEMERAL_ACCOUNT_PRIV_KEY";
+const MY_ASSETS_LIST = "MY_ASSETS_LIST";
+
+export const storeEphemeralAccount = async account => {
+  const { privateKey } = account;
+  await _storeData(EPHEMERAL_ACCOUNT_PRIV_KEY, privateKey);
+};
+
+export const retrieveEphemeralAccount = async () => {
+  let account = null;
+
+  const privateKey = await _retrieveData(EPHEMERAL_ACCOUNT_PRIV_KEY);
+  if (privateKey != null) account = createFromPrivateKey(privateKey);
+
+  return account;
+};
+
+export const storeMyAssets = async myAssets => {
```

Comment:
I wonder if it's overkill to encrypt the assets? Should we use [`AsyncStorage`](https://docs.expo.io/versions/latest/react-native/asyncstorage/) as opposed to [`SecureStore`](https://docs.expo.io/versions/latest/sdk/securestore/) for my assets, most likely for performance reasons?

Reply:
Oh, you are right. Done.

Reply:
Great, thanks. Resolving.